java/oauth-custom-challengehandler/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
+    <string name="app_name">OAuthCustomChallengeHandler</string>
+    <string name="portal_url"></string>
+    <string name="webmap_id"></string>
```

Comment:
Could we provide a default portal url and map id?

Reply:
I had asked Xueming about it, she said the samples team will provide the values to publish publicly. I used our test portal for it. I don't think we want to pass that on.